supplementary/alica_ros1/alica_tests/src/test/Expr/src/Behaviour/SuccessOnInitBeh3821787310391665935.cpp

```diff
@@ -0,0 +1,12 @@
+#include <alica_tests/Behaviour/SuccessOnInitBeh3821787310391665935.h>
+#include <memory>
+
+/*PROTECTED REGION ID(inccppBC3821787310391665935) ENABLED START*/
+// Add additional includes here
+/*PROTECTED REGION END*/
+
+namespace alica
+{
+// Behaviour:SuccessOnInitBeh
+
+} /* namespace alica */
```

Comment:
What is this file doing? It looks like we can remove it from codegen in another PR

Reply:
why should this file be removed? It is the only file that implements the behaviour.

Reply:
This seems to be the implementation file of the domain condition of the behaviour, not the behaviour itself. Also the file only contains the alica namespace without any implementation. But it will be removed when we remove codegen